samcli/local/docker/lambda_image.py

```diff
@@ -288,3 +294,22 @@ def _generate_dockerfile(base_image, layers):
         for layer in layers:
             dockerfile_content = dockerfile_content + f"ADD {layer.name} {LambdaImage._LAYERS_DIR}\n"
         return dockerfile_content
+
+    def _remove_rapid_images(self, repo: str) -> None:
+        """
+        Remove all rapid images for given repo
+
+        Parameters
+        ----------
+        repo st
+            Repo for which rapid images will be removed
+        """
+        LOG.info("Removing rapid images for repo %s", repo)
+        try:
+            for image in self.docker_client.images.list(name=repo):
+                for tag in image.tags:
+                    if ContainerManager.is_rapid_image(tag):
+                        self.docker_client.images.remove(image=image.id, force=True)
```

Comment:
Is force=True safe here?

Reply:
I think it might not be necessary anymore. I'll remove it. @mgrandis let me know if you think this is still needed